payloads/library/general/DuckyTemplate/payload.txt

```diff
@@ -1,18 +1,19 @@
-LED G
+LED SETUP
 ATTACKMODE HID STORAGE
 
 # Set your language here
-LANGUAGE='us'
+DUCKY_LANG=us
```

Comment:
This should be `DUCK_LANG us` instead as you are calling the DUCK_LANG extension and not trying to set a variable yourself.